assets/script.js

```diff
@@ -124,3 +124,24 @@ function searchCard() {
     }
   }
 }
+
+$(document).ready(function() {
```

Comment:
You do not need the document ready block. You can remove the two functions inside of it outside and delete this line and it's corresponding closing tags